### DISCUSSION TITLE
GuildaDev/core-utils

### DISCUSSION THREAD

**Discussion by alexcastrodev** [GuildaDev/toolkit#1] [Q&A]
Hey everyone!

I'm excited to introduce Core Utils, the very first open-source project by Guildadev. First things first, all discussions and issues will be in English.

Our plan is to release our first package in two ways: through a CDN and via npm. The first option allows for standalone usage, while the second one will bundle all the core utilities together.

If you have any questions, feel free to ask right here. And if you're interested in getting involved, you can open an issue or even submit a pull request.

if you don't know the purpose, you can check on: https://github.com/GuildaDev/core-utils

We're currently working on building our documentation, and we'll be sharing how you can contribute soon. Stay tuned! 🚀

**Comment by alexcastrodev** [comment]
The first question here is:

Will we call @guildadev/core-utils ? it was generate by ChatGPT, so you can suggest one for us :)

**Comment by ogrodev** [comment]
I think we can name it something that will imprint our way of doing stuff... doing some kind of joke that would stick to every1 minds

- @guildadev/JediScript
- @guildadev/HiDash - its like lodash but with higher standards
- @guildadev/GTK - Guilda Tool Kit

Something amongst those lines...

**Reply by RodrigoRVSN** [reply]
I liked the `gtk` one

**Reply by alexcastrodev** [reply]
I like linux also LOL
https://www.gtk.org/

**Reply by alexcastrodev** [reply]
what about @guildadev/toolkit ?

**Reply by ogrodev** [reply]
I'm good with `toolkit`, maybe @guildadev/js-tk | @guildadev/jstk dunno

**Reply by alexcastrodev** [reply]
I also like @guildadev/jstk, the only think i wonder is if people will understand what jstk or js-tk means in first glance.
The @guildadev/js-toolkit also can be more explicit.

In the end, `@guildadev/toolkit` and `@guildadev/jstk` works for me.

let's choose between both of that ?

**Reply by ogrodev** [reply]
I'm gonna vote for `@guildadev/toolkit` at then end
